CLAUDE.md

```
@@ -434,39 +434,113 @@ Output a SINGLE-LINE commit subject only:
 
 ## Pull Requests
 
-Use conventional commit format for PR title (no scope):
+Use scoped conventional commit format for PR titles:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)